Codes/Clean Data/DataProperties_20200318.R

```diff
@@ -0,0 +1,115 @@
+rm(list=ls())
+library(data.table)
+library(bizdays)
+library(moments)
+
+directory1<-"W:/LOBSTER/quotes"
+directory2<-"W:/LOBSTER/allMessages"
+directory3<-"W:/LOBSTER/merged"
+# Trading hours (start & end)
+startTrad=9.5*60*60;       # 9:30:00.000 in ms after midnight
+endTrad=16*60*60;        # 16:00:00.000 in ms after midnight
+
+#get list of all files in /quotes/ and /allMessages/ directories
+folders1<-list.files(directory1)
+folders2<-list.files(directory2)
+#get list of tickets
+firms<-vapply(strsplit(folders1,"_"),`[`,1,FUN.VALUE=character(1))
+#get list of dates
+dates<-as.POSIXlt(seq(as.Date("2020-02-01"), as.Date("2020-02-28"),"days"),format="%Y-%m-%d")
+dates<-dates[!dates$wday%in%c(6,0)] #get rid of weekdays
+dates<-dates[dates!=as.POSIXlt(as.Date("2020-02-17"),format="%Y-%m-%d")] #get rid of President's Day

```

Comment:
The problem here is with the output. Without explicitly defining which days are to be included, it is difficult to know the dimensions of the array to output the summary statistics. However, since I am putting the summary statistics to a different code, in this case you're right and I don't need to explicitly define the dates.